multidim_threshold/projection.py

```diff
@@ -54,6 +61,21 @@ def generate_proj_vecs(lo, hi, direc=None):
         yield from vecs
         vecs = [ProjVec(r, direc) for r in next_roots(lo, hi, vecs)]
 
+def generate_axes_intersects(lo, hi, member_oracles, *, searches=None):
+    lo, hi = mdt.map_array((lo, hi))
+    direc = mdt.basis_vecs(len(lo))
+    mids = np.array([fn.nth(0, generate_projections(lo, hi, member_oracles, direc=dir)) for dir in direc])
+    return [axes_mid[:,i].min() for i, axes_mid in enumerate(mids)]
```

Comment:
So `mids` is a 2d numpy array?

I think you can just do:

`return mids.min(axis=1)`

or something. It's been a while for me